static/app/views/settings/account/apiNewToken.tsx

```diff
@@ -19,115 +20,118 @@ import TextBlock from 'sentry/views/settings/components/text/textBlock';
 import PermissionSelection from 'sentry/views/settings/organizationDeveloperSettings/permissionSelection';
 
 const API_INDEX_ROUTE = '/settings/account/api/auth-tokens/';
-type State = {
-  name: string | null;
-  newToken: NewInternalAppApiToken | null;
-  permissions: Permissions;
-};
 
-export default class ApiNewToken extends Component<{}, State> {
-  constructor(props: {}) {
-    super(props);
-    this.state = {
-      name: null,
-      permissions: {
-        Event: 'no-access',
-        Team: 'no-access',
-        Member: 'no-access',
-        Project: 'no-access',
-        Release: 'no-access',
-        Organization: 'no-access',
-        Alerts: 'no-access',
-      },
-      newToken: null,
-    };
-  }
+export default function ApiNewToken() {
+  const [, setName] = useState<string | null>(null);
```

Comment:
we use it later in the code: on line 105

 ```onChange={setName}```

Reply:
we might not need to call `setName` later on because we're never reading the state: `const [, setName] =`

As long as the form submits the input then this `useState` doesn't seem to do anything.

Reply:
ah that's true